server/public/model/feature_flags.go

```diff
@@ -82,7 +82,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.WebSocketEventScope = true
 	f.NotificationMonitoring = true
 	f.ExperimentalAuditSettingsSystemConsoleUI = false
-	f.ExperimentalCrossTeamSearch = false
+	f.ExperimentalCrossTeamSearch = true
```

Comment:
I thought about removing the "Experimental" part of the name but that would require a new release of mobile which currently depends on this name 😅